docs/make.jl

```diff
@@ -1,4 +1,4 @@
-# push!(LOAD_PATH,"../src/")
```

Comment:
The auto documentation for github appears to be broken. I am guessing that it has something to do with the updated folder structure. My initial thought was that your fix here should resolve the issue. This should be investigated and fixed prior to merging with the master branch.